src/pages/Experiments.pug

```diff
@@ -8,35 +8,36 @@ block headVars
-        This information was crucial for building hardware that can attach a roughly equal amount of each nucleotide, following the principle of: “As little as possible and as much as necessary“.
-        This means that the goal is to find conditions where each strand is elongated as little as possible, but with at least one nucleotide in each cycle (see [modeling](https://2021.igem.org/Team:Aachen/Modeling)).
+        This information was crucial for building the hardware that can attach a roughly equal amount of each nucleotide, following the principle of: “As little as possible and as much as necessary“.
+        Thus the goal is to find conditions where each strand is elongated as little as possible, but with at least one nucleotide in each cycle (see [modeling](https://2021.igem.org/Team:Aachen/Modeling)).
         For this purpose, the length distribution of the synthesized fragments was analyzed by [capillary electrophoresis](#CapillaryElectrophoresis).
         
-        Furthermore, different TdTs were tested.
+        Furthermore, enzymes from different manufacturers were tested.
         In the first experiments, we used a recombinant TdT from NEB, but all other experiments were performed with a recombinant TdT from Thermo Fischer Scientific.
-        They use very different reaction buffers, which significantly changed the incorporation speed and nucleotide preferences of TdT.
-        All our summarized results concern the Thermo Fisher TdT.
+        The two manufacturers use very different reaction buffers, which significantly changed the incorporation speed and nucleotide preferences of TdT.
+        We kept working with the Thermo Fisher TdT, thus all our summarized results concern the Thermo Fisher TdT.
         
         # Cyclic Synthesis and Primer Immobilization
-        A cyclic synthesis in solution was tried by just adding the next type of nucleotide to the solution after each elongation.
-        We used a higher primer concentration than in our standard reaction and increased the nucleotide concentration in each reaction step.
+        We have established a cyclic synthesis in solution by adding the next type of nucleotide after each elongation.
+        The primer concentration was higher than in the standard reaction thus every primer gets elongated. Additionally the nucleotide concentration was increased in each reaction step.
         
-        For the semi-specific synthesis, the TdT reaction in solution has the decisive disadvantage that the different nucleotides mix after the first few reaction cycles because there still is a nucleotide excess after each reaction step with that specific nucleotide.
+        The disadvantage of the synthesis reaction in solution is that the different nucleotides mix after the first few reaction cycles because there still are still excess nucleotides which were not incorporated in the previous reaction step. 
```

Comment:
```suggestion
        The disadvantage of the synthesis reaction in solution is that the different nucleotides mix after the first few reaction cycles because there are excess nucleotides, which were not incorporated in the previous reaction step. 
```